content/docs/1.11.0/monitoring/disk-heath.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Disk Health Monitoring
+weight: 4
+---
+
+## Disk Health Metrics
+
+Starting with Longhorn v1.11.0, disk health monitoring metrics are available for both V1 and V2 data engines. These metrics provide insights into disk health status.
+
+> **Note**
+>   - Health data collection periodically (every 10 minutes) by Longhorn's disk monitor.
+>   - Some virtualized or cloud environments (e.g., AWS EBS) may not expose full SMART data, resulting in zero values for certain attributes.
+>   - Available health attributes may vary depending on the underlying disk type and hardware.
+>   - The full set of collected of health data is available in the `nodes.longhorn.io` custom resources (CRs).
+
+### Data Sources
+
+- **V1 Data Engine**:
+    - Health data is collected using SMART monitoring tool (`smartctl`).
+- **V2 Data Engine**:
+    - **NVMe disks:** Health data is retrieved through SPDK.
+    - **AIO disks:** Health data is collected using SMART monitoring tool (`smartctl`).
```

Comment:
Because the data is not consistent across the different types, it may be helpful to mention this and add relevant references, allowing users to verify the information themselves. WDYT?

Reply:
Sounds good! I’ve added it, thank you!